src/screens/SendCoinsConfirmScreen.tsx

```diff
@@ -114,30 +112,20 @@ class SendCoinsConfirmScreen extends Component<Props> {
             type: MessageType.success,
             buttonProps: {
               title: i18n.message.returnToDashboard,
-              onPress: () => this.props.navigation.navigate(Route.MainCardStackNavigator),
+              onPress: () => navigation.navigate(Route.MainCardStackNavigator),
```

Comment:
please move it to class method

Reply:
will be fixed in the optimize wallets fetching PR